client/lib/react-player/CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to ReactPlayer
```

Comment:
I think some of these files were accidentally copied over from this repo https://github.com/CookPete/react-player